ynr/apps/people/tests/test_person_update.py

```diff
@@ -524,11 +523,11 @@ def test_biography_last_updated_timestamp(self):
         """Test that the biography (aka statement to voters)
         shows the last updated timestamp and it
         is updated when the biography is updated"""
-        response = self.app.get(
+        req = self.app.get(
```

Comment:
`app.get` returns a `Response` object, so makes more sense to call it `response` than `req`. To make the name clearer you could go for something like `person_update_response`. The response that you need to update is the one that comes from `/person/{}/` rather than `/person/{}/update`. So the response on line 538 you could call `person_response_one` and then you could make the same call on line 560 and call the response `person_response_two`. Alternatively call them all `response` and just accept it changes through the test. In any case you need something along the lines of 
```
  response = self.app.get(
            "/person/{}/".format(self.person.pk), user=self.user
        )
```
before you do the second check of the last time the statement was updated.